components/Step6/Step6.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {View} from 'react-native';
+import {paths, wizard} from '../../utils/constants';
+import {selectWizard} from '../../redux/reducers/WizardData';
+import {useSelector} from 'react-redux';
+import Button from '../Button/Button';
+import {useMutation} from 'react-query';
+import {saveData} from '../../utils/firebaseUtils';
+import {firebase} from '@react-native-firebase/auth';
+
+import globalStyle from '../../assets/styles/globalStyle';
+
+const Step6 = () => {
+  const wizardData = useSelector(selectWizard);
+  const user = firebase.auth().currentUser;
+  const uid = user ? user.uid : '';
+
+  const {mutate: mutateSaveFitnessData} = useMutation({
+    mutationFn: async (data: {
+      name: string;
+      age: string;
+      weight: string;
+      height: string;
+      level: string;
+      goal: string;
+      workouts: Array<{label: string; name: string}>;
```

Comment:
Use `interface` and `type`. I was expecting from you to notice this by now, these are huge chunks of types inside business logic which makes it unreadable.